client/verta/verta/client.py

```diff
@@ -1238,8 +1239,9 @@ def download_endpoint_manifest(
             Path of the endpoint.
         name : str
             Name of the endpoint.
-        strategy : :ref:`update strategy <update-stategies>`
+        strategy : :ref:`update strategy <update-stategies>`, optional
             Strategy (direct or canary) for updating the endpoint.
+            Default value: DirectUpdateStrategy()
```

Comment:
I think the format for default value in docstring looks like this:
https://github.com/VertaAI/modeldb/blob/master/client/verta/verta/_registry/modelversion.py#L212

Reply:
Thank you. Changed both